migrations/0039-add-feedback-event-field.ts

```diff
@@ -0,0 +1,55 @@
+import { MigrationInterface, QueryRunner, TableColumn, TableIndex, TableForeignKey } from 'typeorm';
+
+const TABLE_NAME = 'Feedback';
+const OLD_NAME = 'title';
+const NEW_NAME = 'source';
+
+export class AddFeedbackEventField1709112961573 implements MigrationInterface {
```

Comment:
Nit: instead of `field`, use `column` in the name of the migration

Reply:
Addressed